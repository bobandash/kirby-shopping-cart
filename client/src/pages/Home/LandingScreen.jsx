import KirbyLogo from "../../assets/kirby-name-logo.png";
import NoTrailStars from "../../assets/no trail.png";
import TrailStars from "../../assets/trail.png";
import styles from "./LandingScreen.module.css";
import KirbySitting from "../../assets/kirby-sitting.png";
import waddleDee from "../../assets/Landing Page/waddle dee.png";
import metaKnight from "../../assets/Landing Page/meta knight.png";
import kingDeDeDe from "../../assets/Landing Page/king de de de.png";

function LandingScreen() {
  return (
    <section className={styles["landing-screen"]}>
      <div className={styles.container}>
        <div className={styles["stars-container"]}>
          <img
            className={styles.stars}
            src={NoTrailStars}
            alt="stars no trail"
          />
          <img
            className={`${styles.stars} ${styles["trail-animation"]}`}
            src={TrailStars}
            alt="stars trail"
          />
        </div>
        <div className={styles["store-information"]}>
          <img
            className={styles["kirby-logo"]}
            src={KirbyLogo}
            alt="kirby-logo"
          />
          <h2 className={styles.subheader}>The Fake Online Store</h2>
        </div>
        <img
          className={styles["kirby-sitting"]}
          src={KirbySitting}
          alt="kirby-sitting"
        />
        <img
          className={`${styles["desktop-image"]} ${styles["top-middle"]}`}
          src={kingDeDeDe}
          alt="king de de de"
        />
        <img
          className={`${styles["desktop-image"]} ${styles["top-left"]}`}
          src={waddleDee}
          alt="waddle dee"
        />
        <img
          className={`${styles["desktop-image"]} ${styles["top-right"]}`}
          src={metaKnight}
          alt="meta knight"
        />
      </div>
    </section>
  );
}

export default LandingScreen;
