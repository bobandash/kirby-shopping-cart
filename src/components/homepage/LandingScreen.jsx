import KirbyLogo from '../../assets/kirby-name-logo.png';
import NoTrailStars from '../../assets/no trail.png';
import TrailStars from '../../assets/trail.png';
import styles from './LandingScreen.module.css';


function LandingScreen(){
  return(
    <section className = {styles["landing-screen"]}>
       <div className = {styles.container}>
          <img className = {styles.stars} src = {NoTrailStars} alt = "stars no trail" />
          <img className = {`${styles.stars} ${styles["trail-animation"]}`} src = {TrailStars} alt = "stars trail" />    
          <div className = {styles["store-information"]}>
            <img className = {styles["kirby-logo"]} src = {KirbyLogo} alt = "kirby-logo" />
            <h2 className = {styles.subheader}>The Fake Online Store</h2>
          </div>
       </div>
    </section>
  )
}

export default LandingScreen;