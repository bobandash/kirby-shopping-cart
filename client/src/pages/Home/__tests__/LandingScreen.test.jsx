import LandingScreen from "../LandingScreen";
import { render } from "@testing-library/react";
import { describe, it, expect } from "vitest";

describe("Landing Screen Component", () => {
  it("renders correctly", () => {
    const { container } = render(<LandingScreen />);
    expect(container).toMatchSnapshot();
  });
});
