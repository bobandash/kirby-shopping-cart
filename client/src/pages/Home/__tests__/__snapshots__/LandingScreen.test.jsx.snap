// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Landing Screen Component > renders correctly 1`] = `
<div>
  <section
    class="_landing-screen_b2b0af"
  >
    <div
      class="_container_b2b0af"
    >
      <div
        class="_stars-container_b2b0af"
      >
        <img
          alt="stars no trail"
          class="_stars_b2b0af"
          src="/src/assets/no trail.png"
        />
        <img
          alt="stars trail"
          class="_stars_b2b0af _trail-animation_b2b0af"
          src="/src/assets/trail.png"
        />
      </div>
      <div
        class="_store-information_b2b0af"
      >
        <img
          alt="kirby-logo"
          class="_kirby-logo_b2b0af"
          src="/src/assets/kirby-name-logo.png"
        />
        <h2
          class="_subheader_b2b0af"
        >
          The Fake Online Store
        </h2>
      </div>
      <img
        alt="kirby-sitting"
        class="_kirby-sitting_b2b0af"
        src="/src/assets/kirby-sitting.png"
      />
      <img
        alt="king de de de"
        class="_desktop-image_b2b0af _top-middle_b2b0af"
        src="/src/assets/Landing Page/king de de de.png"
      />
      <img
        alt="waddle dee"
        class="_desktop-image_b2b0af _top-left_b2b0af"
        src="/src/assets/Landing Page/waddle dee.png"
      />
      <img
        alt="meta knight"
        class="_desktop-image_b2b0af _top-right_b2b0af"
        src="/src/assets/Landing Page/meta knight.png"
      />
    </div>
  </section>
</div>
`;
